### DISCUSSION TITLE
当Tiddler中文本过长时出现的一个问题

### DISCUSSION THREAD

**Discussion by Strelitzia-Reginae-Aiton** [tiddly-gittly/TidGi-Desktop#208] [Q&A]
Tiddler类型markdown，当tiddler内文本过长时（在实际使用中，tiddler内字数在两万以上时，在tiddler后面的一部分会出现问题，出现问题的起始字数尚不明确），光标所在位置与实际输入的位置会有1~2行之间的偏差。
当Tiddler内部文本超过12万时基本不能够通过Tidgi对tiddler进行编辑（会出现光标位置与实际编辑位置不一致与拖动滚动条时卡顿的问题），但是预览依然流畅。(网页端的编辑与预览都很流畅)。
这是NodeJS版TiddlyWiki本身的问题，还是因为插件对页面渲染的影响？

**Comment by linonetwo** [comment]
编辑模式下可能是 codemirror enhanced 插件的问题吧，它要做很多解析可能影响了性能。改成原生文本框编辑器可能会好点儿，还有文本太长就不叫 tiddler 了…

**Reply by linonetwo** [reply]
之后改用 https://github.com/tiddly-gittly/slate-write 可能会好点儿
